threadsd/main.go

```diff
@@ -118,6 +118,7 @@ func main() {
 			return true
 		}),
 		grpcweb.WithWebsockets(true),
+		grpcweb.WithWebsocketPingInterval(time.Second*5),
```

Comment:
Can you make this a flag? Should be configurable by the user

Reply:
Sure